src/test/java/fi/helsinki/cs/tmc/core/commands/SubmitTest.java

```diff
@@ -39,12 +39,15 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.net.URI;
 import java.net.URISyntaxException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.text.ParseException;
 import java.util.ArrayList;
 import java.util.List;
+import org.mockito.Matchers;
+import org.mockito.Mockito;
```

Comment:
Imports should be ordered as follows, with groups separated by empty lines:

1) All static imports in a single group
2) `fi.helsinki` imports
3) Third-party imports, one group per top-level package, in ASCII sort order. For example: `android`, `com`, `junit`, `org`, `sun`
4) `java` imports
5) `javax` imports